Library/Formula/ddar.rb

```diff
@@ -8,11 +8,14 @@ class Ddar < Formula
   head "https://github.com/basak/ddar.git"
 
   depends_on "xmltoman" => :build
-  depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on :python
```

Comment:
And, apologies if I'm making this difficult. This python stuff is complicated, and I clearly don't understand it or the Homebrew-Python interaction as well as folks who've been here longer.